Vagrantfile

```diff
@@ -1,10 +1,7 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
```

Comment:
Some cleanup and also bumping up the computing power for faster builds and no more crashing on test runs.